arch/inst/C/c.ldsp.yaml

```diff
@@ -6,26 +6,34 @@ name: c.ldsp
 long_name: Load doubleword from stack pointer
 description: |
   C.LDSP is an RV64C/RV128C-only instruction that loads a 64-bit value from memory
-  into register rd.
+  into register xd.
   It computes its effective address by adding the zero-extended offset, scaled by 8,
   to the stack pointer, x2.
-  It expands to `ld` `rd, offset(x2)`.
-  C.LDSP is only valid when rd &ne; x0 the code points with rd=x0 are reserved.
+  It expands to `ld xd, offset(x2)`.
+  C.LDSP is only valid when xd ≠ x0; code points with xd=x0 are reserved.
 definedBy:
   anyOf:
     - C
     - Zca
-base: 64
-assembly: xd, imm
+assembly: xd, imm(sp)
```

Comment:
remove "(sp)". (This was just removed in a recent commit, also, and just shouldn't be here.)